lib/internal/http.js

```diff
@@ -38,7 +39,7 @@ function getNextTraceEventId() {
 }
 
 function isTraceHTTPEnabled() {
-  return isTraceCategoryEnabled('node.http');
+  return tracing.tracingCategories[0] > 0;
```

Comment:
It's faster to `const { tracingCategories } = internalBinding('tracing');` at the top and then here:
```suggestion
  return tracingCategories[0] > 0;
```
I can go into detail as to why but basically V8 has to do less work this way. It won't make a major difference but death by a thousand cuts, right?

Reply:
When this gets deserialized we need to access it dynamically here, or refresh it upon deserialization. I don't think it's that much of a performance difference here to access it dynamically though it's better to add a comment here in case someone tries to do what's suggested above in the future (which would break deserialization).